### PR TITLE
Update instructions on border routers

### PR DESCRIPTION
**Pull request by MarceloSalazar**
Please find here a proposal to improve the docs.

@SeppoTakalo please have a look and comment whether you are happy with this.

After this, maybe @peknis01 can do sanity checks?

### CODE REVIEW

**Comment by SeppoTakalo** [comment]
Good. I'm happy with these.

**Comment by MarceloSalazar** [comment]
@peknis01 are you ok with this?

**Comment by peknis01** [comment]
Please read through once more and merge if you are ok with my edits.

File: README.md [hunk 1]
```diff
@@ -151,7 +153,7 @@ To change the RF driver:
...
+mbed OS provides you with total control of the device. However, some defaults are always loaded if you do not provide proper information regarding them. This becomes evident when you switch among platforms. On some platforms, a particular pin might be reserved for a particular functionality (depending upon the MCU) which thus cannot be used generally. If the Ardurino form factor is not available, or the required peripherial is on a different pin, the pins values can be set manually. For example, you can add `"atmel-rf.spi-rst": "D4"` to your `mbed_app.json` file. This will set the SPI_RST pin to D4 of the GPIO.
```
Comment (MarceloSalazar) [inline README.md]: > However, some defaults are always loaded if you do not provide proper information regarding them. This becomes evident when you switch among platforms.

Can we elaborate a bit on this area? I'm not sure what defaults (configuration?) are you referring to.

Other minor typos:
Ardurino --> Arduino 
peripherial --> peripheral

**Comment by peknis01** [comment]
@SeppoTakalo or @hasnainvirk , please elaborate the text as suggested by Marcelo.

**Comment by MarceloSalazar** [comment]
@hasnainvirk would you have chance to elaborate a bit more on the suggested sentence?

**Comment by MarceloSalazar** [comment]
Issue fixed here https://github.com/ARMmbed/oob-mbed-os-example-mesh-minimal/pull/11

**Pull request merged**